src/main/scripts/org/reaktivity/specification/mqtt/connection/client.rpt

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect await ROUTED_SERVER
+        "nukleus://streams/mqtt#0"
+  option nukleus:window 8192
+  option nukleus:transmission "duplex"
+
+connected
+
+write [0x10 0x10]           # fixed header
+      [0x00 0x04] "MQTT"    # protocol name
+      [0x05]                # protocol version
+      [0x02]                # connect flags
+      [0x00 0x3C]           # keep alive
```

Comment:
Please include the value of keep alive in seconds in the comment.
Please apply these changes across all similar files.